v22.1/import.md

```diff
@@ -8,24 +8,20 @@ docs_area: reference.sql
 
 The `IMPORT` [statement](sql-statements.html) imports the following types of data into CockroachDB:
 
-- [Avro][avro]
```

Comment:
Should we add a section saying Avro, CSV/TSV and Delimited are supported using IMPORT INTO and link to that page? I find it slightly disconcerting that our import landing page only show two formats.

Reply:
> Should we add a section saying Avro, CSV/TSV and Delimited are supported using IMPORT INTO and link to that page? I find it slightly disconcerting that our import landing page only show two formats.

Yes, definitely. This was present in the warning in an "included" file. But I was only using that in this one place, so have put this inline for now. Lmk if you think this is enough of a push toward finding those formats on the `IMPORT INTO` page.